testing/matrix-sdk-integration-testing/src/tests/timeline.rs

```diff
@@ -873,6 +873,13 @@ async fn test_new_users_first_messages_dont_warn_about_insecure_device_if_it_is_
     // But when alice becomes cross-signed and bob finds out about it
     cross_sign(&alice).await;
     bob.sync_once(SyncSettings::new()).await.expect("should not fail to sync");
+
+    // Sync again to make sure the server has notified us about the update to
+    // alice's device info.
+    bob.sync_once(SyncSettings::new().timeout(Duration::from_millis(2000)))
```

Comment:
Ah nice, I thought you'd be using `tokio::time::timeout` but using the timeout from `SyncSettings` is even better if it achieves the same effect :+1:

Reply:
It seems to work...